### ISSUE TITLE
Orientation attribute is None for Sequence objects

### ISSUE THREAD

**Issue by handstad** [mdshw5/pyfaidx#80]
Using pyfaidx 0.4.5.2, the orientation is None if the Sequence object is not reverse complement:

```python
from pyfaidx import Fasta
f = Fasta("/path/to/human_g1k_v37_decoy.fasta", one_based_attributes=False)
s = f[1][10000:10005]
src = -s
scomp = s.complement

s.orientation # Wrongly returns None, not 1
src.orientation # Wrongly returns None, not -1
scomp.orientation # Returns -1 (is this correct?)
```

**Comment by mdshw5** [comment]
I'm closing this issue as it's fixed and actually related to #79..

**Comment by huguesfontenelle** [comment]
with pyfaidx==0.4.6 (latest),
`s.orientation` still returns `None`.
while
`scomp.orientation` returns `-1`.
#79

**Comment by mdshw5** [comment]
Fixed in 0.4.7.1
